packages/main/src/SuggestionItem.js

```diff
@@ -83,7 +83,7 @@ const metadata = {
 		 * Defines the state of the <code>info</code>.
 		 * <br><br>
 		 * Available options are: <code>"None"</code> (by default), <code>"Success"</code>, <code>"Warning"</code> and <code>"Erorr"</code>.
-		 * @type {string}
+		 * @type {ValueState}
```

Comment:
can you also add the @ defaultvalue None

and below for the "group" @ defaultvalue dalse

Reply:
done